official/bert/benchmark/bert_benchmark.py

```diff
@@ -54,7 +55,7 @@ def __init__(self, output_dir=None):
     self.num_steps_per_epoch = None
 
   @flagsaver.flagsaver
-  def _run_bert_classifier(self, callbacks=None):
+  def _run_bert_classifier(self, callbacks=None, use_ds=True, enable_xla=False):
```

Comment:
How is a user going to turn on XLA in the actual code?  This will make it hard to debug and we really cannot have other users try this out or get the benefit.  Maybe if we added a TODO(b/) so we know this needs rolled into the model.